scripts/release/socket.sh

```diff
@@ -12,15 +12,17 @@ fi
 
 INSTANCE=$1
 gpgp=$(find /usr/lib/gnupg{2,,1} -type f -name gpg-preset-passphrase 2> /dev/null)
-KEYGRIP=$(gpg -K --with-keygrip --textmode dev@algorand.com | grep Keygrip | head -1 | awk '{ print $3 }')
+#KEYGRIP=$(gpg -K --with-keygrip --textmode dev@algorand.com | grep Keygrip | head -1 | awk '{ print $3 }')
```

Comment:
this file doesn't looks right.

Reply:
Yeah, one of the problems I found is that the keygrip, which is generated from the public key:
```
To identify a key we use a thing called keygrip which is the SHA-1 hash of an canonical encoded S-Expression of the public key as used in Libgcrypt. For the purpose of this interface the keygrip is given as a hex string. The advantage of using this and not the hash of a certificate is that it will be possible to use the same keypair for different protocols, thereby saving space on the token used to keep the secret keys. 
```
https://www.gnupg.org/documentation/manuals/gnupg/Agent-Protocol.html

If a key has a subkey, then there will be more than one keygrip.  I've decided to hardcode the keygrips in for now, and the reason I didn't remove or change the code yet is that I would like to revisit this.

Since there is going to be at least one more iteration of the release build before pencils down, I thought it would be ok to leave as-is.